### ISSUE TITLE
Is this possible to use on the client side?

### ISSUE THREAD

**Issue by arhoy** [t3chnoboy/amazon-product-api#120]
INside the src folder of a Create React App I have created a file called Amazon with following code.

```
import amazon from "amazon-product-api";

const client = amazon.createClient({
  awsId: "",
  awsSecret: "",
  awsTag: ""
});

export const amazonItemSearch = async () => {
  const data = await client.itemSearch({
    director: "Quentin Tarantino",
    actor: "Samuel L. Jackson",
    searchIndex: "DVD",
    audienceRating: "R",
    responseGroup: "ItemAttributes,Offers,Images"
  });

  return data;
};
```

However, I am running into an error saying
Access to fetch at 'https://webservices.amazon.com/ 
 'http://localhost:3000' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.

How can i use this library to make calls on the client side? I have tried to proxy the URL https://webservices.amazon.com/ without luck

**Comment by JuanCrg90** [comment]
You can't use this package in the web app, you can read more about this in the following issues. https://github.com/t3chnoboy/amazon-product-api/issues/66 https://github.com/t3chnoboy/amazon-product-api/issues/86

**Comment by arhoy** [comment]
I see okay thank you! Is it possible there is a package you can use?

**Comment by JuanCrg90** [comment]
I don't think so. The product API requires some delicate API keys that you shouldn't expose in your front end application.
